src/modules/expression/binop/lt.rs

```diff
@@ -1,15 +1,17 @@
 use heraclitus_compiler::prelude::*;
 use crate::docs::module::DocumentationModule;
+use crate::handle_binop;
+use crate::modules::expression::expr::Expr;
 use crate::translate::compute::{translate_computation, ArithOp};
 use crate::utils::{ParserMetadata, TranslateMetadata};
 use crate::translate::module::TranslateModule;
-use super::{super::expr::Expr, parse_left_expr, expression_arms_of_type};
 use crate::modules::types::{Typed, Type};
+use super::BinOp;
 
 #[derive(Debug, Clone)]
 pub struct Lt {
-    left: Box<Expr>,
-    right: Box<Expr>
+    pub left: Box<Expr>,
+    pub right: Box<Expr>
```

Comment:
Where do you need it to be public?

Reply:
This is a deprecated change. I’ll revert these and fields for other modules back to private! Thanks